### PR TITLE
Add missing labels in VS ontology

### PR DESCRIPTION
**Pull request by kwlodarczyk0**
Signed-off-by: kwlodarczyk0 <wlodarczyk.krystian00@gmail.com>

Fixes: #118

### CODE REVIEW

File: VS/VehicleSignals.rdf [hunk 1]
```diff
@@ -873,6 +874,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CatalystTemperatureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Catalyst Temperature Sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:877]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 2]
```diff
@@ -1023,6 +1025,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CoolantTemperatureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Coolant Temperature Sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:1028]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 3]
```diff
@@ -1033,6 +1036,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CrankshaftPositionSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Crankshaft Position Sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:1039]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 4]
```diff
@@ -1226,6 +1230,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CushionPositionSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Cushion Position Sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:1233]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

**Review by mereolog — CHANGES_REQUESTED** [review]

**Review by mskulim — APPROVED** [review]

**Pull request merged**